lib/gatewayService.js

```diff
@@ -621,7 +625,7 @@ function GatewayService(options) {
         sendTransaction: sendTransaction,
         performTransaction: performTransaction,
         performTargetedTransaction: performTargetedTransaction,
-        GenerateXsell: GenerateXsell,
+        generateXsell: generateXsell,
```

Comment:
This looks like a breaking change no? 
Could we keep `GenerateXsell` as an alias?
```
        GenerateXsell: generateXsell,
        generateXsell: generateXsell,
```

Reply:
This is not a change I would want to push upstream to RG since it is a new addition to their code

Reply:
this is not breaking change, because it is not even exposed in the public API... in the index file. it is only used in the test file, which i updated too.